modules/core/src/main/java/org/apache/synapse/config/xml/EnrichMediatorFactory.java

```diff
@@ -32,8 +38,12 @@
 
 import java.util.Properties;
 
-
+/**
+ * Factory for {@link EnrichMediator} instances.
+ */
 public class EnrichMediatorFactory extends AbstractMediatorFactory {
+    private static final Log logger = LogFactory.getLog(EnrichMediatorFactory.class.getName());
```

Comment:
What is the usage of logger variable? if there is no usage shall we remove that variable?